hazelcast/src/main/java/com/hazelcast/map/impl/MerkleTreeNodeEntries.java

```diff
@@ -55,11 +54,11 @@ public void setNodeOrder(int nodeOrder) {
         this.nodeOrder = nodeOrder;
     }
 
-    public Set<EntryView<Data, Data>> getNodeEntries() {
+    public Set<WanMapEntryView<Object, Object>> getNodeEntries() {
         return nodeEntries;
     }
 
-    public void setNodeEntries(Set<EntryView<Data, Data>> nodeEntries) {
+    public void setNodeEntries(Set<WanMapEntryView<Object, Object>> nodeEntries) {
```

Comment:
unused method along with `getNodeOrder`, `setNodeOrder`?

Reply:
Yes, looks like it.